helm/values.yaml

```diff
@@ -0,0 +1,18 @@
+domain:
+  url: platform.debezium.io
+stage:
+  image: quay.io/debezium/platform-stage:latest
+api:
+  image: quay.io/debezium/platform-conductor:nightly
+kafka:
+  enabled: true
```

Comment:
TBH I'm not sure whether we should pull in Strimzi --  I get the idea of ease of use, but it still leaves us with the need to manage dependency and I don't like tat -- considering it's just one of many sinks

Reply:
Yeah, I was not sure if adding it or not. Anyway this also depends on https://issues.redhat.com/browse/DBZ-8417
since if we for example decide for a specif offset/history storage you should have the option to install it as dependency. 

As of now we can just remove it and install as an external step for the example. WDYT?

Reply:
Agreed. We should also figure out how to do configuration.
Either k8 config plugin or we just mount a config map as `application.yml`.

As for offsets -- I think that we should currently support two options. In-Memory and Redis. Configurable only for the whole deployment. WDYT?

Reply:
> The envs are then provided in the values.yaml.

How? Not sure what you mean.  ALSO in general I am not particularly a fan of configuring via env vars. The names get ridiculously long. But initially it's fine I guess.

Reply:
Something like this https://stackoverflow.com/a/60098256/24117340

The only configuration here that I think it's relevant to change is the one associated to the offsets and the one about the database that we should put in a secret.